.ci/openshift-ci/test.sh

```diff
@@ -65,44 +66,44 @@ END
 }
 
 deployChe() {
-  chectl server:deploy  --che-operator-cr-patch-yaml=custom-resources.yaml --telemetry=off --platform=openshift --installer=operator --batch
+  chectl server:deploy --che-operator-cr-patch-yaml=custom-resources.yaml --telemetry=off --platform=openshift --installer=operator --batch
 }
 
 patchTestPodConfig(){
-  # obtain the basic "happy-path" pod config
-  cat .ci/openshift-ci/happy-path-pod.yaml > happy-path-pod.yaml
+  # obtain the basic test pod config
+  cat .ci/openshift-ci/plugins-test-pod.yaml > plugins-test-pod.yaml
```

Comment:
could we just grab the pod definition from che e2e repository and just update config map with the tests to launch ?

Reply:
The "plugin-test-pod.yaml" has defined exactly for che-plugin-registry set of parameters.
As you can see we patch only product url which defined in runtime.

Reply:
but we don't need to have 64 lines of yaml just to change product URL
all the settings are default happy path configuration that are tied to che/e2e, not this repository

Reply:
it looks like script should be in https://github.com/eclipse/che/tree/master/tests/.infra or https://github.com/eclipse/che/tree/master/tests/e2e/files/happy-path

Reply:
We need it for defining all these parameters.
Besides, please pat attention that these tests have no relations to the "Happy-Path"
```
      env:
        - name: TS_SELENIUM_OCP_USERNAME
          value: 'user'
        - name: TS_SELENIUM_OCP_PASSWORD
          value: 'user'
        - name: TS_SELENIUM_VALUE_OPENSHIFT_OAUTH
          value: 'true'
        - name: TS_SELENIUM_USERNAME
          value: 'admin'
        - name: TS_SELENIUM_PASSWORD
          value: 'admin'
        - name: TS_OCP_LOGIN_PAGE_PROVIDER_TITLE
          value: 'htpasswd'
        - name: TEST_SUITE
          value: test-all-plugins
        - name: NODE_TLS_REJECT_UNAUTHORIZED
          value: '0'
        - name: TS_SELENIUM_BASE_URL
          value: CHE_URL
        - name: TS_SELENIUM_LOG_LEVEL
          value: DEBUG
        - name: TS_SELENIUM_MULTIUSER
          value: 'true'
        - name: TS_SELENIUM_VALUE_TLS_SUPPORT
          value: 'true'
        - name: TS_SELENIUM_W3C_CHROME_OPTION
          value: 'true'          
      volumeMounts:
        - name: test-run-results
          mountPath: /tmp/e2e/report/
      resources:
        requests:
          memory: "8Gi"
          cpu: "2000m"
        limits:
          memory: "8Gi"
          cpu: "4000m" 
```

Reply:
but it looks like this one: https://github.com/eclipse/che/blob/f8a7aaef6b7db0b59bef382f5eefaac204236ca0/tests/performance/pod-oauth.yaml

nothing it tied to plugin registry in the values you reference

Reply:
The example you provided has a different set of parameters. Besides this file in the Che repo because the "load-tests" rely on che. Let me explain my opinion. I am trying to explain that file we discussing for is specific for che-plugin-registry , so I don't want to keep something specific for one repo in the common "e2e" repository. I'll prefer to store specific files in the specific repo.

Reply:
all the env value except `test-all-plugins` look like default properties when you want to test Eclipse Che.
so all these defaults should be in e2e/tests repository

it's like
```
volumeMounts:
        - name: test-run-results
          mountPath: /tmp/e2e/report/
```

it's not related to plug-in registry as well